tests/test_sample.py

```diff
@@ -26,6 +26,7 @@ def test_positive_login_1(self):
 
         @Tags: t1, t2, t3
 
+        @Types: Functional
```

Comment:
This is for a negative test, it is ok as the output expect it to be on the skipped lines.